### ISSUE TITLE
maestral gui unexpected keyword argument 'include_non_downloadable_files'

### ISSUE THREAD

**Issue by Alessandro-Barbieri** [SamSchott/maestral-dropbox#33]
It crashes with

```
 > Loading access token...
 > MaestralClient is ready.
Up to date
Maestral index could not be found. Rebuild if necessary.
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/maestral/gui/first_sync_dialog.py", line 200, in on_dropbox_path
    self.populate_folders_list()
  File "/usr/local/lib/python3.7/dist-packages/maestral/gui/first_sync_dialog.py", line 241, in populate_folders_list
    root_folders = self.mdbx.client.list_folder("", recursive=False)
  File "/usr/local/lib/python3.7/dist-packages/maestral/client.py", line 476, in list_folder
    **kwargs
TypeError: files_list_folder() got an unexpected keyword argument 'include_non_downloadable_files'
```

**Comment by SamSchott** [comment]
Apologies, this is an oversight in the requirements. Maestral needs the latest version of the python Dropbox SDK (v9.4.0). Could you update by running `python3 -m pip install --upgrade dropbox`?

**Comment by Alessandro-Barbieri** [comment]
ok it works

**Comment by SamSchott** [comment]
Alright, good. I’ll leave the issue open until the next version is released, just in case others have the same problem.

**Comment by SamSchott** [comment]
Fixed in  v0.2.4
